src/style.css

```
@@ -1 +1,20 @@
 /* styles go here */
+.form-field {
+  display: block;
+}
+
+.mb {
```

```suggestion
.mb {
```
Такі назви класів, зазвичай, використовуються в css-бібліотеках. Давай краще використаємо більш інтутивну назву